src/test/java/plugins/WarningsPluginTest.java

```diff
@@ -86,6 +82,289 @@
             + "Priority test = Priority.NORMAL;\n"
             + "return new Warning(all, 42, all, all, all);";
 
+    public static final String RESOURCE_WARNING_MAIN_JAVA = "WarningMain.java";
+    public static final String RESOURCE_WARNING_MAIN_JAVA_PATH = "/warnings_plugin/"
+                                                                + RESOURCE_WARNING_MAIN_JAVA;
+
+    public static final String CMD_WARNING_MAIN_JAVA_CONSOLE = "javac -Xlint:all "
+                                                                + RESOURCE_WARNING_MAIN_JAVA;
+
+    public static final String RESOURCE_CODE_NARC_REPORT = "CodeNarcXmlReport.xml";
+    public static final String RESOURCE_CODE_NARC_REPORT_PATH = "/warnings_plugin/jenkins-17787/"
+                                                                    + RESOURCE_CODE_NARC_REPORT;
+
+
+
+    @Inject
+    SshKeyPair keyPair;
+
+    @Inject
+    private DockerContainerHolder<JavaContainer> dockerContainer;
+
+    @Inject
+    private SshdContainer sshdDocker;
+
+    @WithDocker
+    @Test
+    public void detailsTabContentTest() throws ExecutionException, InterruptedException {
```

Comment:
I thought there are already enough tests **without** docker. We wrote some different scenarios to ensure, that it works also with docker.